physionet-django/project/forms.py

```diff
@@ -772,8 +770,8 @@ def __init__(self, editable=True, **kwargs):
         licenses = License.objects.filter(
             resource_types__icontains=str(self.instance.resource_type.id))
         # Set allowed access policies based on license policies
-        available_policies = [a for a in range(len(Metadata.ACCESS_POLICIES)) if licenses.filter(access_policy=a)]
-        self.fields['access_policy'].choices = tuple(Metadata.ACCESS_POLICIES[p] for p in available_policies)
+        available_policies = [a for a in range(len(ACCESS_POLICIES)) if licenses.filter(access_policy=a)]
```

Comment:
This will fail if somebody decides to change the definition of `ACCESS_POLICIES` to something like:
```
    ACCESS_POLICIES = (
        (2, 'Open'),
        (4, 'Restricted'),
        (6, 'Credentialed'),
    )
```
I know there's no reason to, but it would be safer it this line of code doesn't assume that the first element is also an index of that element in the collection. So I would propose something like that:
```suggestion
        available_policies = [policy[0] for policy in ACCESS_POLICIES if licenses.filter(access_policy=policy[0]).exists()]
```
Additionally .exists() might be a bit more efficient than evaluation to bool in the if statement.

Reply:
Updated. The old code was indeed very strange.